### ISSUE TITLE
Enhance a11y for `Tabs`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#7603]
name: Tabs

### Description
1 accessibility violation was found in the `Tabs`.

#### [Components / Tabs / Dynamic](https://yamada-ui.github.io/yamada-ui/?path=/story/components-tabs--dynamic)

Received:
Certain ARIA roles must contain particular children (aria-required-children)

Fix any of the following:
  Element has children which are not allowed: button

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.13/aria-required-children?application=axeAPI

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.

**Comment by iimtii** [comment]
@hirotomoyamada can I take this issue?

**Comment by hirotomoyamada** [comment]
@iimtii 

Hi, thanks for the comment.

Yes, pls.
